bascule/basculehttp/metrics.go

```diff
@@ -0,0 +1,48 @@
+package basculehttp
+
+import (
+	"net/http"
+
+	"github.com/Comcast/comcast-bascule/bascule"
+)
+
+type Monitor interface {
```

Comment:
This is really a more general Listener abstraction.  Client code can get notified of authentications.  I would consider switching to this:

```
type Listener interface {
    OnAuthenticated(bascule.Authentication)
}
```

Then, remove any mention of metrics here.  It's just a general purpose listener that a client can plug metrics into.